AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -94,6 +95,9 @@ class CompleteScreenViewModel: ObservableObject {
     }
     
     func selectedStream(with id: Int) {
+        if let stream = externalSessionWithStreams?.streams.first(where: { $0.id == id }) {
```

Comment:
The symbol below the chart doesn't change. I think this method should look something like this:
```
if let stream = externalSessionWithStreams?.streams.first(where: { $0.id == id }) {
            assignValues(with: stream)
            defineChartRange(with: stream)
        }
```